src/Microsoft.AspNetCore.JsonPatch/JsonPatchDocumentOfT.cs

```diff
@@ -93,7 +93,7 @@ public JsonPatchDocument<TModel> Add<TProp>(Expression<Func<TModel, TProp>> path
 
             Operations.Add(new Operation<TModel>(
                 "add",
-                ExpressionHelpers.GetPath(path).ToLowerInvariant() + "/" + position,
+                ExpressionHelpers.GetPath(path, ContractResolver).ToLowerInvariant() + "/" + position,
```

Comment:
No, its being used only in this class as of now. Are you suggesting a name change or make it stateful type by supplying the resolver?